stages/10-openebs-install-check/openebs-install-check

```diff
@@ -23,6 +23,10 @@ kubectl get po -n openebs
 ##  Sequencing and Running test   ##
 ####################################
 
+######################################## Applying e2e-CRD
+echo "***Applying e2e-crd***********"
+kubectl apply -f utils/e2e-crd.yml
```

Comment:
`e2e-crd` is already present in the cluster. It's being applied here - https://github.com/mayadata-io/oep-e2e-gcp/blob/d27a84a2ffdcc1a8f2881238fcec0de959511fdc/stages/5-cluster-connect-check/create-apikey-check#L16